### ISSUE TITLE
Should `?exploreNumbers` be public?

### ISSUE THREAD

**Issue by arouinfar** [phetsims/make-a-ten#308]
We recently updated the Teacher Tips template to include common code query parameters (https://github.com/phetsims/query-string-machine/issues/52) and QA is working on updating all Teacher Tips (https://github.com/phetsims/qa/issues/963). Part of this process involved QA reviewing all sims to see if there are public query parameters that should be included in the Teacher Tips.

The `exploreNumbers` query parameter is not currently public, but looks like something that would be helpful to teachers. From what I can tell, it was introduced in https://github.com/phetsims/make-a-ten/issues/151 to make it easier to test the limit of 9999 on the Explore Screen. It prepopulates the Explore screen with the specified numbers, for example:
https://phet.colorado.edu/sims/html/make-a-ten/latest/make-a-ten_all.html?exploreNumbers=99,236

The query parameter is generally well-behaved in the latest published version and on master. It has validation to make sure no individual number exceeds 9999. However, there isn't any validation that the sum of the numbers respects this maximum, so the sim can start to behave strangely if you try something like:
https://phet.colorado.edu/sims/html/make-a-ten/latest/make-a-ten_all.html?exploreNumbers=9999,9999,9999,9999

At some point in the future, this sim will be republished with the various updates introduced in counting-common. The design team should consider if `exploreNumbers` should be public-facing, and if so, if any further validation is necessary. 

Since this sim isn't currently being worked on, I'll tag @amanda-phet @jonathanolson @mariahmaephet, but won't assign.

**Comment by samreid** [comment]
@catherinecarter do you want to do anything about this issue for the 1.1 release?

**Comment by catherinecarter** [comment]
I don't think we should make this public. Using the values, 9999 and 236, the sum is 10,235. When I add 1 to 9999, the sim breaks since the total is higher than the allowable max (similar to issue #299). 

So I say we don't make this public as it will create an additional bug that we don't need to willingly introduce.

**Comment by samreid** [comment]
Sounds good, it is already non-public so this issue seems good to close.
